deno/lib/ZodError.ts

```diff
@@ -356,6 +364,11 @@ export const defaultErrorMap = (
     case ZodIssueCode.not_multiple_of:
       message = `Number must be a multiple of ${issue.multipleOf}`;
       break;
+    case ZodIssueCode.invalid_file_type:
+      message = `The file type should be ${issue.expected.join(
+        " or "
+      )}. Received: ${issue.received}`;
+      break;
```

Comment:
Missing here messages for `too_big` and `too_small`. Without it for `file` the error goes to the generic:
![image](https://user-images.githubusercontent.com/49499/158582704-42cbfd4e-3cfa-4d97-bfbf-43b59ac1673b.png)

Reply:
I can do this changes if you give me push permission to your branch. If not is fine I'll wait :)